plugins/tekton/package.json

```diff
@@ -39,10 +40,11 @@
     "@material-ui/lab": "^4.0.0-alpha.45",
     "@patternfly/patternfly": "^5.1.0",
     "@patternfly/react-charts": "^7.1.1",
-    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-core": "^5.1.2",
     "@patternfly/react-icons": "^5.1.1",
     "@patternfly/react-tokens": "^5.1.2",
     "@patternfly/react-topology": "^5.1.0",
+    "@patternfly/react-table": "^5.1.2",
```

Comment:
It could go into @aonic-ui as well but the reason for this is to keep here is because most of the PF packages are here and being used as a peer dependency in aonic. So the library doesn't ship any of the pf packages in it.

Reply:
@aonic-ui/pipelines 1.0.1 package now brings in this dependency, so I have updated this PR to use the latest version and remove react-table from here.

Reply:
Thanks Karthik!!